main.js

```diff
@@ -0,0 +1,204 @@
+let weatherData = [];
+let weekData = {};
+
+const DateTime = luxon.DateTime;
+
+const addData = (data) => {
+  weatherData = [];
+  let weatherFetched = {
+    weather: Math.floor(data.main.temp),
+    city: data.name,
+    condition: data.weather[0].description,
+    icon: 'http://openweathermap.org/img/w/' + data.weather[0].icon + '.png'
+  }
+  weatherData.push(weatherFetched);
+  renderWeather(weatherData);
+};
+
+let forecastData = [];
```

Comment:
I'd keep this in the same spot as your other global variables. Also, it's not advised that you use global variables.